### ISSUE TITLE
Invalid video in documentation

### ISSUE THREAD

**Issue by duraki** [RhinoSecurityLabs/SleuthQL#7]
From the README (and official [web](https://rhinosecuritylabs.com/application-security/sleuthql-sql-injection-discovery-tool/)), we are getting: 

`This video is no longer available because the uploader has closed their YouTube account.`

Can we get reupload or something?

Thanks.

**Comment by codingo** [comment]
Agreed - video was a useful primer for this. Would be great to see it re-posted.

**Comment by duraki** [comment]
Paging @djhohnstein to take a look into it

**Comment by djhohnstein** [comment]
Hey guys,

Sorry it took me awhile to get back to you. I no longer am a maintainer of these repositories so I haven't been watching them closely. The video still exists and just needs to be dug up by the new maintainers and re-uploaded (as I no longer have the MP4). The commands should still be valid though from the official web version as you guys have linked above.

The issue has been raised to the requisite parties and they'll be handling the issue from here on forward. 

Cheers!

**Comment by codingo** [comment]
Any chance you could put the command string here?

**Comment by djhohnstein** [comment]
From the article [here](https://rhinosecuritylabs.com/application-security/sleuthql-sql-injection-discovery-tool/) (https://rhinosecuritylabs.com/application-security/sleuthql-sql-injection-discovery-tool/), scroll down to "Using SleuthQL". Save the selected items from your site. Then, you can use the tool as the [Readme states](https://github.com/RhinoSecurityLabs/SleuthQL#usage) (https://github.com/RhinoSecurityLabs/SleuthQL#usage) by pointing the script to the XML, along with passing a comma separated list of domains that you want to analyze. This will create a directory for each domain with request text files.

To run SQLMap in batch, use the `find` command pointing it to the directory of requests files and run SQLMap in batch via:
```bash
find $directoryDomainName -name “*.txt” -exec sqlmap -r {} –batch \;
```

**Comment by codingo** [comment]
Thank-you @djhohnstein. The command string I was referring to though I bleive was a bit more detailed t o allow quicker loading of targets? It's been a while since I made the slide deck for the  talk this is in and unfortunately I didn't put the command there.

**Comment by djhohnstein** [comment]
If I were to _guess_ you might be thinking about Burp Spidering which will automatically crawl the site and submit forms. Otherwise, if you wanted to check the results of your SQLMap you can check the log file for successes.

**Comment by codingo** [comment]
Closed with https://github.com/RhinoSecurityLabs/SleuthQL/commit/29fc878813e651c883168afe8c9ffb424806775d
